test/test_ops.py

```diff
@@ -737,4 +737,15 @@ def _case_five_transform(t):
 instantiate_device_type_tests(TestCommon, globals())
 
 if __name__ == '__main__':
+    if IS_PYTORCH_CI:
+        # Assure no opinfo entry has dynamic_dtypes
+        filtered_ops = list(filter(opinfo_helper.is_dynamic_dtype_set, op_db))
+        for op in filtered_ops:
+            fmt_str = opinfo_helper.str_format_dynamic_dtype(op)
+            print(fmt_str)
```

Comment:
I like what's happening here but wonder if we can provide clearer context to the user by first printing the context (like below), then printing each operator that needs an update and what the recommended update is.

Reply:
Makes sense.

Do you know any way to make sure this will run after `run_tests` or after the last test?
(If we place it after run_tests it doesn't run)

Idea is to (like you mentioned previously), error out after all the tests so that we also get signal on other failed tests.

Reply:
That's tricky and I don't know a great way to do this offhand. Maybe we should consider not asserting but simply printing out the warning and the change, and then try to enforce the policy of not accepting OpInfos with their dtypes set dynamically?

Reply:
> then try to enforce the policy of not accepting OpInfos with their dtypes set dynamically?

Where can we do that? As a test? But then the point of failure and point of warning would differ.

Reply:
With `tearDownClass`,

CI Log : https://github.com/pytorch/pytorch/runs/2721694549
```
======================================================================
ERROR [0.000s]: tearDownClass (__main__.TestOpInfoCPU)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_ops.py", line 41, in tearDownClass
    assert len(filtered_ops) == 0, err_msg
AssertionError: The operator/s below is using dynamic_dtypes in the OpInfo entry!Please set the dtypes manually

        OpInfo(atan2,
               dtypes=all_types_and(torch.bool,),
               dtypesIfCUDA=(),
        )
        

        OpInfo(topk,
               dtypes=all_types(),
               dtypesIfCUDA=(),
        )
```